### PR TITLE
Gui leaderboard

### PR DESCRIPTION
**Pull request by ngdao**
Fixed stat display
Added leader board function

### CODE REVIEW

**Comment by mshannahnv** [comment]
This looks good on my end!

File: gui/src/main/java/DiceGameGUI.form [hunk 1]
```diff
@@ -0,0 +1,313 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```
Comment (anderspitman) [inline gui/src/main/java/DiceGameGUI.form:1]: Is this file supposed to be in git? We didn't need it before.

Comment (ngdao) [inline gui/src/main/java/DiceGameGUI.form:1]: It helps with NetBeans Design window, not necessary for building, can be ignored

Comment (anderspitman) [inline gui/src/main/java/DiceGameGUI.form:1]: Ok just wanted to make sure you needed it.

**Pull request merged**